metadata_mapper/lambda_function.py

```diff
@@ -49,34 +53,30 @@ def parse_enrichment_url(enrichment_url):
 # {"collection_id": 26098, "mapper_type": "nuxeo", "page_filename": 2}
 # AWS Lambda entry point
 def map_page(payload, context):
-    if settings.LOCAL_RUN:
+    if settings.LOCAL_RUN and not isinstance(payload, dict):
         payload = json.loads(payload)
 
     vernacular_reader = import_vernacular_reader(payload.get('mapper_type'))
     source_vernacular = vernacular_reader(payload)
     api_resp = source_vernacular.get_api_response()
-    source_metadata_records = source_vernacular.parse(api_resp)
+    mapped_records = source_vernacular.parse(api_resp)
```

Comment:
These records shouldn't be mapped yet at this point. They should be parsed from the stashed API response, but not yet mapped. There are a few enrichment processes that run prior to mapping.

Reply:
Reverted